src/main/scala/com/danielasfregola/twitter4s/http/clients/Client.scala

```diff
@@ -18,7 +18,7 @@ trait Client extends OAuthClient {
   protected def sendAndReceive[T](request: HttpRequest, f: HttpResponse => Future[T])(
       implicit system: ActorSystem,
       materializer: Materializer): Future[T] = {
-    implicit val _ = request
+    implicit val r: HttpRequest = request
```

Comment:
Underscore (`_`) is no longer a valid identifier for `val`s, so we have to give it a valid name (even if it's just `r`) - see also scala/bug#10384, scala/bug#7691 & scala/scala#6218.

Reply:
oh nice! I didb't know :)